libsolidity/ast/AST.h

```diff
@@ -723,6 +721,42 @@ class VariableDeclaration: public Declaration
 	Location m_location; ///< Location of the variable if it is of reference type.
 };
 
+class ModifierArea: public ASTNode
+{
+public:
```

Comment:
I think indentation here is off.

Reply:
I just copied it from surrounding classes